WallPaperDev/WallPaperDev/Views/DetailGoal Views/CheckBoxButton.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  CheckBoxButton.swift
+//  WallPaperDev
+//
+//  Created by Jackson Ho on 4/15/20.
+//  Copyright © 2020 Stephen Ouyang. All rights reserved.
+//
+
+import UIKit
+
+class CheckBoxButton: UIButton {
+  override init(frame: CGRect) {
+    super.init(frame: frame)
+    layer.borderColor = ApplicationDependency.manager.currentTheme.colors.lightGray.cgColor
+    layer.borderWidth = 1.5
+    layer.cornerRadius = 5
+    let checkMark = UIImage(systemName: "checkmark")?.withRenderingMode(.alwaysTemplate)
```

Comment:
Dope! How are you liking SFSymbols? :D

Reply:
Personally wished they used an enum based system or something of the sorts instead of just a string which is far easier to have a typo for 🤦

Reply:
SF symbols so far has been great for providing template icons for the app without having to add another assets to the project.